client/src/stories/Button.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Button from '../components/Button';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Buttons',
+};
+export const loginButton = () => (
+  <Button onClick={action('link to Login')}>Login</Button>
+);
+
+export const createButton = () => (
```

Comment:
I would call it `createAccountButton`, since `createButton` can mean a lot of stuff. Just a small thing =)

Reply:
You're right, thank you for the advice 👍🏼